ProtoZBuffer/ProtoZBuffer.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>ProtoZBuffer</id>
+    <version>1.0.0.0</version>
+    <title>ProtoZBuffer</title>
+    <authors>CAST</authors>
+    <owners>CAST</owners>
+    <licenseUrl>http://LICENSE_URL_HERE_OR_DELETE_THIS_LINE</licenseUrl>

```

Comment:
We should complete the information. But I did not know under which licence ProtozBuffer was provided...
Still in progress :D